inbm/dispatcher-agent/dispatcher/fota/fota.py

```diff
@@ -153,9 +153,7 @@ def trigger_reboot() -> None:
                 state = {'restart_reason': "pota"}
                 dispatcher_state.write_dispatcher_state_to_state_file(state)
                 logger.debug(status)
-                return_message = COMMAND_SUCCESS
-                self._dispatcher_callbacks.broker_core.send_result(status)
-                self._dispatcher_callbacks.broker_core.send_result(str(return_message))
+                self._dispatcher_callbacks.broker_core.telemetry(status)
```

Comment:
Should we be updating the state file, when we aren't restarting yet?  I thought about removing those changes also.

Reply:
Since it's been tested already as it is and I gave the system back over to validation, unless there is a serious issue with leaving it, I'll probably leave it as it is here.